sql/i4_invalid_indexes.sql

```diff
@@ -13,7 +13,14 @@ select
     coalesce(nullif(pn.nspname, 'public') || '.', '') || pct.relname as "relation_name",
     pci.relname as index_name,
     pn.nspname as schema_name,
-    pct.relname as table_name
+    pct.relname as table_name,
+    pg_size_pretty(pg_relation_size(pidx.indexrelid)) index_size,
+    format('DROP INDEX CONCURRENTLY %s; -- %s, table %s', pidx.indexrelid::regclass::text, 'Invalid index', pct.relname) as drop_code,
```

Comment:
please use multiple lines here to avoid too wide lines:
```
format(
  'DROP INDEX CONCURRENTLY %s; -- %s, table %s', pidx.indexrelid::regclass::text,
  'Invalid index',
  pct.relname
) as drop_code,
```